packages/website/src/pages/styles.module.css

```diff
@@ -33,6 +33,20 @@
   margin: 1rem 1rem;
 }
 
+.buttonPrimary {
+  background-color: var(--ifm-color-warning-dark);
+  color: var(--ifm-color-secondary-contrast-foreground);
```

Comment:
😕 The gold is pretty jarring given that it's the only not-blue/purple on the page. I have to admit, I'm not a fan of the visual. Could you switch this to just having a thick outline/shadow as suggested by the issue?

Reply:
@JoshuaKGoldberg Thanks for reviewing it! 
Of course, I'll make this change as soon as I can.